src/applications/mhv-landing-page/components/alerts/AlertMhvUserAction.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+// eslint-disable-next-line import/no-named-default
+import { default as recordEventFn } from '~/platform/monitoring/record-event';
+
+const AlertMhvUserAction = ({ errorCode, testId, recordEvent }) => {
+  const headline = `Error code ${errorCode}: Contact the My HealtheVet help desk`;
+  useEffect(
+    () => {
+      recordEvent({
+        event: 'nav-alert-box-load',
+        action: 'load',
+        'alert-box-headline': headline,
+        'alert-box-status': 'warning',
+      });
+    },
+    [headline, recordEvent],
+  );
+
+  return (
+    <div
+      className="mhv-c-reg-alert mhv-u-reg-alert-error usa-alert usa-alert-error vads-u-display--flex
+        vads-u-align-items--flex-start vads-u-justify-content--center vads-u-flex-direction--row
+        vads-u-margin-bottom--3"
+      data-testid={testId}
+    >
```

Comment:
Can we use the VSDS `VaAlert` [component](https://design.va.gov/storybook/?path=/docs/uswds-va-alert--docs#error) instead of custom alert here? This will fix the existing icon color & size issue.
eg:  
```
<VaAlert data-testid={testId} status="error" visible>
...existing content...
</VaAlert>
`